src/ray/protobuf/common.proto

```diff
@@ -343,6 +343,8 @@ message ActorDiedErrorContext {
   // Whether the actor had never started running before it died, i.e. it was cancelled
   // before scheduling had completed.
   bool never_started = 10;
+  // Whether the actor was on a preempted node.
+  bool preempted = 11;
```

Comment:
I think we can actually just insert node death protobuf here if it is due to node death?